doc/install_mne_c.rst

```diff
@@ -167,6 +167,29 @@ listed in :ref:`CIHDGFAA`.
     |                         | values, see :ref:`mne_process_raw`.        |
     +-------------------------+--------------------------------------------+
 
+
+Troobleshooting
+###############
+
+Missing libXp.so.6
+^^^^^^^^^^^^^^^^^^
+
+If when running an MNE command you get an error about a missing library ``libXp.so.6`` such as:
+
+.. code-block:: console
+
+    $ mne_process_raw
+    mne_process_raw: error while loading shared libraries: libXp.so.6: cannot open shared object file: No such file or directory
+
+If you are using Ubuntu Linux and you have admin access to the machine you can do:
+
+.. code-block:: console
+
+    $ sudo add-apt-repository "deb http://security.ubuntu.com/ubuntu precise-security main"
```

Comment:
it's probably eaiser just to do a `wget` on the `.deb` and a `sudo apt install ...`, but if this works it's probably fine, too.

Also I think there is often a missing `libgfortran.so.3` (?) that could be listed here, does it get pulled in as a dependency by this? If not we can deal with it next time someone brings it up.